wire-ios/Wire-iOS/Sources/Authentication/Interface/ViewControllers/AuthenticationCredentialsViewController.swift

```diff
@@ -432,6 +437,19 @@ final class AuthenticationCredentialsViewController: AuthenticationStepControlle
         return true
     }
 
+    func textFieldShouldBeginEditing(_ textField: UITextField) -> Bool {
+        guard
+            textField == emailInputField,
+            isRegistering,
+            let prefilledCredentials,
+            prefilledCredentials.credentials.emailAddress != nil
+        else {
+            return true
+        }
+
+        return false
```

Comment:
We don't want to allow editing the prefilled credentials because then the user may change registration to an email that it's not allowed to (e.g the domain is claimed).